mlrun/api/utils/clients/iguazio.py

```diff
@@ -466,11 +478,17 @@ def _generate_auth_info_from_session_verification_response(
         if planes:
             planes = planes.split(",")
         planes = planes or []
+        user_unix_id = None
+        x_unix_uid = response.headers.get("x-unix-uid")
+        if x_unix_uid and x_unix_uid.lower() != "unknown":
+            user_unix_id = int(x_unix_uid)
+
```

Comment:
In what case `x_unix_uid = 'uknown'`? I think that requires a comment

Reply:
In your comment you are referring to `Unknown` while in the validation you are validating `unknown`.